android-app/app/src/main/java/net/trustly/inappbrowserandroid/MainActivity.kt

```diff
@@ -44,13 +44,13 @@ class MainActivity : AppCompatActivity() {
                 return true
             }
         }
-        webView.loadUrl("http://localhost:3000?integrationContext=InAppBrowser&urlScheme=in-app-browser-android")
+        webView.loadUrl("http://192.168.1.2:3000?integrationContext=InAppBrowser&urlScheme=in-app-browser-android")
```

Comment:
@andresguedes It's not a big deal, but maybe, change the localhost, or a local IP to the sandbox url as a default, I think that can help merchant to not confuse the local IP with some Trustly url.